.eslintrc

```diff
@@ -1,10 +0,0 @@
----
```

Comment:
I ended up moving this to a `.js` file as I just don't understand yaml 🙈 Somehow I got lost in objects within arrays and eslint just didn't want to do what I wanted it to do.